packages/malte-common/src/oauth/AuthError.ts

```diff
@@ -0,0 +1,4 @@
+export enum AuthError {
+  auth_needed,
```

Comment:
enums in typescript are usually PascalCase: https://www.typescriptlang.org/docs/handbook/enums.html